app/src/main/java/pl/edu/zut/mad/appzut/fragments/WebPlanFragment.java

```diff
@@ -0,0 +1,148 @@
+package pl.edu.zut.mad.appzut.fragments;
+
+import android.annotation.SuppressLint;
+import android.graphics.Bitmap;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.JavascriptInterface;
+import android.webkit.WebResourceResponse;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.Toast;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import pl.edu.zut.mad.appzut.R;
+import pl.edu.zut.mad.appzut.network.DataLoadingManager;
+import pl.edu.zut.mad.appzut.network.ScheduleEdzLoader;
+
+public class WebPlanFragment extends Fragment {
+
+    @BindView(R.id.web_view_plan)
+    WebView web;
+
+    @BindView(R.id.please_wait)
+    View pleaseWaitView;
+
+    public static final String USERNAME_ARG = "username";
+    public static final String PASSWORD_ARG = "password";
+
+    private String username;
+    private String password;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_web_plan, container, false);
+        ButterKnife.bind(this, view);
+        username = getArguments().getString(USERNAME_ARG);
+        password = getArguments().getString(PASSWORD_ARG);
+        return view;
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        initWebView();
+    }
+
+    @SuppressLint("AddJavascriptInterface")
+    private void initWebView() {
+        pleaseWaitView.setVisibility(View.VISIBLE);
+        web.getSettings().setJavaScriptEnabled(true);
+        web.addJavascriptInterface(this, "android");
+        web.setWebViewClient(new WebViewClient(){
+            private String mReceivedErrorForUrl;
+
+            @Override
+            public void onPageFinished(WebView view, String url) {
+                web.loadUrl("javascript:javascript:(function(){document.body.appendChild(document.createElement('script')).src='/appwizut-injected-script.js'})()");
+                web.loadUrl("javascript:var x =document.getElementById('ctl00_ctl00_ContentPlaceHolder_MiddleContentPlaceHolder_txtIdent').value = '"+username+"';");
```

Comment:
s/`javascript:var x = document`/`javascript:void document`/ (Avoid extra variable)

Also, I'm pretty sure there will be user that has `'` in password (or `%` used in url-encoding, though I don't know if other things at ZUT break on these cheracters)